pkg/api/v1/types.go

```diff
@@ -175,6 +175,17 @@ type ObjectMeta struct {
 	// queryable and should be preserved when modifying objects.
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/annotations.md
 	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// List of objects depended by this object. If ALL objects in the list have
+	// been deleted, this object will be garbage collected.
+	OwnerReferences []ObjectReference `json:"ownerReferences,omitempty"`
```

Comment:
Please don't use ObjectReference. Create a type that contains exactly the fields we plan to use.